dev-tools/omicron-dev/src/bin/omicron-dev.rs

```diff
@@ -320,6 +332,116 @@ async fn cmd_clickhouse_run(args: &ChRunArgs) -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+async fn start_replicated_cluster() -> Result<(), anyhow::Error> {
+    // Start a stream listening for SIGINT
+    let signals = Signals::new(&[SIGINT]).expect("failed to wait for SIGINT");
+    let mut signal_stream = signals.fuse();
+
+    // Start the database server and keeper processes
+    let cur_dir = std::env::current_dir().unwrap();
+    let replica_config =
+        cur_dir.as_path().join("oximeter/db/src/configs/replica_config.xml");
```

Comment:
It seems like we might want to use the project directory structure here, rather than CWD. E.g, something like [`CARGO_MANIFEST_DIR`](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates), which points to the workspace root. That would allow folks to run this from more than just the project root itself.